tests/cases/compiler/explicitReturnUnknow.ts

```diff
@@ -0,0 +1,2 @@
+function f(): unknown { ""; }
```

Comment:
The new test isn't really necessary because `unknownType1` was already testing this.